.github/workflows/compare.yaml

```diff
@@ -0,0 +1,92 @@
+name: Compare Schemas for Changes
+
+on:
+  push:
```

Comment:
This action probably only makes sense on pull requests, because what really matters is to make sure that a proposed change does not (or does, if it's expected) produce a schema change.